views/partials/functions/pretty.js

```diff
@@ -10,7 +10,7 @@ const bytes = function (a, b) {
       d = b || 2,
       e = ["Bytes", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"],
       f = Math.floor(Math.log(a) / Math.log(c));
-    return parseFloat((a / Math.pow(c, f)).toFixed(d)) + " " + e[f];
+    return parseFloat((a / Math.pow(c, f)).toFixed(d)).toFixed(2) + " " + e[f];
```

Comment:
- first tofixed **string** on division result
- then to float numeric
- back to tofixed string

please clean up ;)